_datasets/septa-alerts.md

```diff
@@ -6,17 +6,17 @@ created: '2014-12-08'
 license: License Not Specified
 maintainer: SEPTA
 maintainer_email: septoid@gmail.com
-maintainer_link: https://wwww.septa.org/open-data/
+maintainer_link: [https://wwww.septa.org/open-data/](https://wwww.septa.org/open-data/)
 maintainer_phone: null
 notes: 'Provides access to SEPTA travel alerts via an API. All travel alerts can be 
-  viewed at https://www3.septa.org/api/Alerts/.<br>
+  viewed at [https://www3.septa.org/api/Alerts/](https://www3.septa.org/api/Alerts/).<br>
```

Comment:
they do work in notes or descriptions but not sure if it makes sense to use them as much